phantazm-neuron-minestom/src/main/java/com/github/phantazmnetwork/neuron/bindings/minestom/entity/config/MinestomDescriptorConfigProcessor.java

```diff
@@ -0,0 +1,39 @@
+package com.github.phantazmnetwork.neuron.bindings.minestom.entity.config;
+
+import com.github.phantazmnetwork.neuron.bindings.minestom.entity.GroundMinestomDescriptor;
+import com.github.phantazmnetwork.neuron.bindings.minestom.entity.MinestomDescriptor;
+import com.github.steanky.ethylene.core.ConfigElement;
+import com.github.steanky.ethylene.core.ConfigPrimitive;
+import com.github.steanky.ethylene.core.collection.ConfigNode;
+import com.github.steanky.ethylene.core.collection.LinkedConfigNode;
+import com.github.steanky.ethylene.core.processor.ConfigProcessException;
+import com.github.steanky.ethylene.core.processor.ConfigProcessor;
+import net.minestom.server.entity.EntityType;
+import org.jetbrains.annotations.NotNull;
+
+public class MinestomDescriptorConfigProcessor implements ConfigProcessor<MinestomDescriptor> {
+
+    @Override
+    public MinestomDescriptor dataFromElement(@NotNull ConfigElement element) throws ConfigProcessException {
+        switch (element.getStringOrThrow("type").toUpperCase()) {
+            case "GROUND" -> {
+                EntityType type = EntityType.fromNamespaceId(element.getStringOrThrow("entityType"));
+                String ID = element.getStringOrThrow("ID");
+                return GroundMinestomDescriptor.of(type, ID);
```

Comment:
there are a lot of other properties configurable using different GroundMinestomDescriptors, should probably add support for those too

Reply:
jumpHeight, fallTolerance, stepHeight, and calculator
should I leave the width depth height alone?

Reply:
width/depth/height should be left along as they (generally) should correspond with the entity's actual width/height/depth